bundler/lib/bundler/compact_index_client/updater.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require_relative "../vendored_fileutils"
+require "rubygems/package"
```

Comment:
can we instead use `gem/package/digest_io` ?

Reply:
unfortunately that causes the error `NameError: uninitialized constant Gem::Package`